src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -82,7 +84,54 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func logPanic(proxyAddr string, msg string) {
+	cfg, err := config.Init(packageVersion)
+	if err != nil {
+		panic("Error initializing config")
+	}
+
+	// Create the client-side proxy.
+	c := &client.Client{
+		// TODO: Next port in case the regular one hasn't been closed properly
+		Addr:         proxyAddr,
+		ReadTimeout:  0, // don't timeout
+		WriteTimeout: 0,
+	}
+	ready := make(chan bool)
+	go c.ListenAndServe(func() { ready <- true })
+	<-ready
+
+	if err := logging.Init(); err != nil {
+		panic("Error initializing logging")
+	}
+
+	ready = logging.Configure(proxyAddr, cfg.CloudConfigCA, cfg.InstanceId, version, revisionDate)
+	<-ready
+
+	log.Error(msg)
```

Comment:
If the message is passed as-is, it will be logged in multiple lines (as multiple messages), because `panic` produces a ton of output. We could do `strings.Replace("\n", "\\n", -1)` to replace newlines with the escaped symbol. Anything better? Some sort of encoding? Replacing with whitespace?